src/global/interfaces/link-button-properties.ts

```diff
@@ -275,3 +275,15 @@ export function forwardHostEvent(
 
   nestedActionElement.dispatchEvent(copiedEvent);
 }
+
+/**
+ * Forwards a change on the nested input element to the host element.
+ * In this way an event triggered in the ShadowDOM can cross its boundary and can be listened on the host component.
+ */
+export function forwardInnerEventToHost(event: Event, host: HTMLElement): void {
+  event.preventDefault();
+  event.stopImmediatePropagation();
```

Comment:
do we need to stop it as it doesn't bubble shadow dom boundaries anyways?. But it's also ok to stop it :)